ui/apps/platform/src/Containers/Policies/Table/PoliciesTable.tsx

```diff
@@ -354,7 +362,7 @@ function PoliciesTable({
                                     isSelected: allRowsSelected,
                                 }}
                             />
-                            {columns.map(({ Header, width }) => {
+                            {/* columns.map(({ Header, width }) => {
```

Comment:
Curious - any reason to keep the commented out code?

Reply:
Good eyes. Let me include add comment here from pull request that added `//` so less mysterious.